### ISSUE TITLE
$q.dark links

### ISSUE THREAD

**Issue by nothingismagick** [quasarframework/quasar-ui-qmarkdown#58]
I had to override the `.q-markdown--link` color with `!important` which is a bit more heavyhanded than I like.

```
.body--dark
  body
    background #606c71
    color #FFFDFA

  a, a.q-markdown--link
    color #00D5ED!important
```

**Comment by hawkeye64** [comment]
@nothingismagick Why not PR this?

**Comment by hawkeye64** [comment]
In the next release, you have control over the color of links, except when in a container.

**Comment by hawkeye64** [comment]
Fix will be in v1.0.10
